extensions/amp-lightbox/1.0/component.jss.js

```diff
@@ -38,39 +38,61 @@ before allowing buttons to be activated. */
 };
 
 const wrapper = {
-  top: 0,
-  bottom: 0,
-  left: 0,
-  right: 0,
-  width: '100%',
-  height: '100%',
-  position: 'fixed',
-  boxSizing: 'border-box',
-};
+  top: '0 !important',
+  left: '0 !important',
+  bottom: '0 !important',
+  right: '0 !important',
+  width: '100% !important',
+  height: '100% !important',
+  position: 'fixed !important',
+  boxSizing: 'border-box !important',
+
+  /** Scroll containment:
+   *  overflow: hidden does not trigger overscrollBehavior, so
+   *  overflow: scroll is applied even though the lightbox should not scroll.
+   *  We rely instead on the overflow: hidden in content layer to prevent
+   *  scrolling inside the lightbox.
+   */
+  overflow: 'scroll !important', // Prevent scrolling inside lightbox.
+  overscrollBehavior: 'none !important', // Prevent scrolling outside lightbox.
 
-// User overridable styles
-const defaultStyles = {
+  // User overridable styles
   zIndex: 1000,
   backgroundColor: 'rgba(0, 0, 0, 0.9)',
   color: '#fff',
 };
 
-/* 
-  overflow: hidden does not trigger overscrollBehavior, so 
-  overflow: auto is applied even though the lightbox should not scroll.
-  We rely instead on the overflow: hidden in ContainWrapper to prevent 
-  scrolling inside the lightbox.
-*/
-const containScroll = {
-  overflow: 'auto', // Prevent scrolling inside lightbox.
-  overscrollBehavior: 'none', // Prevent scrolling outside lightbox.
+const scrollable = {};
+const content = {
+  overflow: 'hidden !important',
+  '&$scrollable': {
+    overflow: 'scroll !important',
```

Comment:
Is there any way to keep it at `auto`? Or does scroll containment not work with `auto`?